src/components/Editor/Footer.js

```diff
@@ -18,7 +18,7 @@ import { isPretty, isLoaded, getFilename } from "../../utils/source";
 import { getGeneratedSource } from "../../reducers/sources";
 import { shouldShowFooter, shouldShowPrettyPrint } from "../../utils/editor";
 
-import PaneToggleButton from "../shared/Button/PaneToggle";
+import { PaneToggleButton } from "../shared/Button";
```

Comment:
Really tempted to go with "Buttons", since the folder contains a number bigger than one.